cli.js

```diff
@@ -142,17 +208,29 @@ async function measureUpload(bytes, iterations) {
   return measurements;
 }
 
+function logInfo(text, data) {
+  console.log(bold(' '.repeat(15 - text.length), `${text}:`, blue(data)));
+}
+
 function logSpeedTestResult(size, test) {
   const speed = stats.median(test).toFixed(2);
   console.log(
-    bold(' '.repeat(7 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
+    bold(' '.repeat(9 - size.length), size, 'speed:', yellow(`${speed} Mbps`))
   );
 }
 
 async function speedTest() {
+  const serverLocationData = await fetchServerLocationData();
+
+  const { ip, loc, colo } = await fetchCfCdnCgiTrace();
+
+  logInfo('Your IP', ip);
+  logInfo('Your Country', loc);
+  logInfo('Server Location', `${serverLocationData[colo]} (${colo})`);
```

Comment:
Maybe move `serverLocationData[colo]` to a variable named `city` or something like that to improve readability

Reply:
LGTM, I will update to that.